definitions/variable/emissions/emissions.yaml

```diff
@@ -3,24 +3,27 @@
 # Gross emissions are only defined for CO2.
 
 - Emissions|{Level-1 Species}:
-    description: Emissions of {Level-1 Species}
+    description: Emissions of {Level-1 Species}, net of emissions to the atmosphere and the removal of {Level-1 Species} from the atmosphere.
     unit: "{Level-1 Species}"
     components:
       - Emissions|{Level-1 Species}|AFOLU
       - Emissions|{Level-1 Species}|Energy
       - Emissions|{Level-1 Species}|Industrial Processes
       - Emissions|{Level-1 Species}|Waste
-      - Emissions|{Level-1 Species}|Capture and Removal
+      - Emissions|{Level-1 Species}|Other Capture and Removal
+      - Emissions|{Level-1 Species}|Product Use
       - Emissions|{Level-1 Species}|Other
 - Gross Emissions|CO2:
-    description: Gross emissions of carbon dioxide (CO2), not accounting for negative
-      emissions from bioenergy with CCS (BECCS) or agriculture, forestry
-      and other land use (AFOLU)
+    description: Gross emissions of carbon dioxide (CO2), not accounting for 
+      emissions removed from the atmosphere
     unit: Mt CO2/yr
 
 - Emissions|{Level-2 Species}|AFOLU:
     description: Emissions of {Level-2 Species} from agriculture, forestry
-      and other land use (IPCC category 3)
+      and other land use (IPCC category 3), reporting the net of emissions 
+      including land-use methods that do not use CCS such as soil carbon 
+      sequestration, biochar, forestry, peatland and wetland restoration, 
+      and biomass burial
```

Comment:
I see that this can be useful clarification, both for the reporting and then for use in climate models afterwards.
Added the note as suggested in 28fcd7a7fe6392884a3d06179423660367a946ce